.github/workflows/aks.yaml

```diff
@@ -29,6 +29,7 @@ env:
   name: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
   location: westeurope
   cost_reduction: --node-vm-size Standard_B2s --node-osdisk-size 30
+  duration: 5
```

Comment:
Nit: the commit title should say `Update GH workflow duration to 5s` and then provide the justification in the body. When I read the gitlog, this commit will show as `Updating GH workflows`, which is not useful to me as a developer digging through git history, which we do often when bisecting failures, etc, so it would force me to look at the diff to make sure I understand what `Updating` meant.

Reply:
ack - this actually will eventually be moved as it was there to reduce the test duration for the performance test. The latest commit changes the behavior a bit -- perf test is only ran if the user explicitly passes `--perf`.

Reply:
I since we are now modifying how we execute the test with `--perf` I will make these cmd line options vs envvars.